tools/gen_selectedops.py

```diff
@@ -61,7 +61,7 @@ def gen_cmake_oplist(opconfig_file, oplist_cmake_file='_selectedoplist.cmake'):
     if len(sys.argv) == 2:
```

Comment:
Please remember to add a mapping to `OPMAP_TO_CMAKE_FLAGS` in this file because we will select operators to use when build together with onnxruntime.